cmd/kubeadm/app/apis/kubeadm/v1beta1/types.go

```diff
@@ -113,6 +115,7 @@ type ClusterConfiguration struct {
 	ClusterName string `json:"clusterName,omitempty"`
 }
 
+// DEPRECATED
```

Comment:
i think we should add the message:
```
DEPRECATED - This group version of X is deprecated by apis/kubeadm/v1beta2/X.
```

to:

InitConfiguration
JoinConfiguration
ClusterConfiguration

but not have it for the other objects.